ftf_cli/commands/register_output_type.py

```
@@ -75,11 +76,21 @@ def register_output_type(yaml_path, profile, inferred_from_module):
                     }
                 )
 
-        # Prepare the request payload
+        # Generate lookup tree from properties
+        try:
+            lookup_tree = properties_to_lookup_tree(output_type_def["properties"])
+            lookup_tree_json = json.dumps(lookup_tree)
+            click.echo(f"✅ Generated lookup tree from properties")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)